com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -766,20 +724,18 @@ internal void Initialize(bool server)
                 }
             }
 
-            NetworkConfig.NetworkTransport.OnTransportEvent += HandleRawTransportPoll;
+            if (!NetworkConfig.NetworkTransport.UseTransportPolling())
```

Comment:
Is this new behaviour? Or was it handled elsewhere and needed to be moved for the connection changes?

Reply:
Well, really it was something I noticed that didn't make sense the way it was organized.  Currently, we both poll and subscribe to events... polling actually calls into NetworkTransport every update.  This change makes handling inbound messages either via polling or event driven... but never both.
It is a subtle change that is "connection related" and I was tossing it out there to see how it was received.
In NetworkConnectionManager.OnEarlyUpdate:
```
        internal void OnEarlyUpdate()
        {
            ProcessPendingApprovals();

            if (NetworkManager.NetworkConfig.NetworkTransport.UseTransportPolling())
            {
#if DEVELOPMENT_BUILD || UNITY_EDITOR
                s_TransportPollMarker.Begin();
#endif
                NetworkEvent networkEvent;
                do
                {
                    networkEvent = NetworkManager.NetworkConfig.NetworkTransport.PollEvent(out ulong clientId, out ArraySegment<byte> payload, out float receiveTime);
                    NetworkManager.HandleRawTransportPoll(networkEvent, clientId, payload, receiveTime);
                    // Only do another iteration if: there are no more messages AND (there is no limit to max events or we have processed less than the maximum)
                } while (NetworkManager.IsListening && networkEvent != NetworkEvent.Nothing);

#if DEVELOPMENT_BUILD || UNITY_EDITOR
                s_TransportPollMarker.End();
#endif
            }
        }
```
If you are not polling it skips that entire block of code, because whole chunk of code doesn't need to be executed if you are not polling and it removes any profile information/tracking when using an event driven transport.

🤷 

Since it all was within NetworkConnectionManager, it seemed like a logical improvement for how we are doing it today.
😸